### PR TITLE
Add new case install_uninstall/downgrade_upgrade driver for virtio keyboard device

### PR DESCRIPTION
**Pull request by peixiu**
Add new case: with_vioinput.device_keyboard..uninstall_install/downgrade_upgrade

id: 1456178 1764447
Signed-off-by: Peixiu Hou <phou@redhat.com>

### CODE REVIEW

**Comment by peixiu** [comment]
(1/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.uninstall_install.q35: PASS (814.16 s)
 (2/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.downgrade_upgrade.q35: PASS (265.63 s)

**Comment by peixiu** [comment]
(1/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.uninstall_install: PASS (228.36 s)
 (2/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.downgrade_upgrade: PASS (259.28 s)

**Comment by peixiu** [comment]
@luckyh Could you help to review this patch? thanks a lot~

File: qemu/tests/cfg/win_virtio_driver_update_test.cfg [hunk 1]
```diff
@@ -177,3 +177,17 @@
...
+            driver_name = "vioinput"
+            device_name = "VirtIO Input Driver"
+            device_hwid = '"PCI\VEN_1AF4&DEV_1052"'
+            inputs = input1
+            input_dev_bus_type_input1 = virtio
+            variants:
+                - device_keyboard:
+                    input_dev_type_input1 = keyboard
+                    run_bgstress = "vioinput_keyboard"
+                    key_table_file = key_to_keycode_win.json
```
Comment (luckyh) [inline qemu/tests/cfg/win_virtio_driver_update_test.cfg]: Please update all the configurations according to the changes of https://github.com/autotest/tp-qemu/pull/1853

Comment (peixiu) [reply qemu/tests/cfg/win_virtio_driver_update_test.cfg]: updated, thanks~

**Comment by peixiu** [comment]
(1/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.uninstall_install: PASS (139.93 s)
 (2/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_virtio_driver_update_test.with_vioinput.device_keyboard.before_bg_test.downgrade_upgrade: PASS (180.05 s)

**Comment by peixiu** [comment]
@luckyh Hi xuhan, I updated the code, could help to review it again?
Thanks a lot~

**Pull request merged**